tests/integration/420.command.init.test.js

```diff
@@ -322,6 +322,110 @@ test('netlify init new Next.js site', async (t) => {
   })
 })
 
+test('netlify init new Next.js site with correct default build directory and build command', async (t) => {
+  const [command, publish, functions] = ['next build', '.next', 'custom-functions']
+  const initQuestions = [
+    {
+      question: 'Create & configure a new site',
+      answer: answerWithValue(DOWN),
+    },
+    { question: 'Team: (Use arrow keys)', answer: CONFIRM },
```

Comment:
[sand maybe dust?] For another PR, but as we expand the test suite, it may be worth looking at making the question texts constants to avoid copy pasta errors or if the question text is ever modified (unlikely?).

Reply:
🍜